hack/dev/gen.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+set -eou pipefail
+
+cd pki
+
+IP_ADDR="${1}"
+CERT_LENGTH=$(( 24 * 365 * 1 ))
+NODE="master-1"
+
+if [[ $(uname -s) == "Linux" ]]; then
```

Comment:
nit, might be worthwhile to allow path to osctl to be specified ( either by cli arg or env var )